src/main/java/org/springframework/samples/petclinic/repository/springdatajpa/SpringDataPetTypeRepositoryImpl.java

```diff
@@ -41,8 +41,8 @@ public class SpringDataPetTypeRepositoryImpl implements PetTypeRepositoryOverrid
 	public void delete(PetType petType) {
         this.em.remove(this.em.contains(petType) ? petType : this.em.merge(petType));
 		Integer petTypeId = petType.getId();
-		
-		List<Pet> pets = this.em.createQuery("SELECT pet FROM Pet pet WHERE type_id=" + petTypeId).getResultList();
```

Comment:
Does this syntax not working any more?

Reply:
Unfortunately that doesn't work anymore,  `ClinicServiceJPATests.shouldDeletePetType` fails with the exception: `org.hibernate.query.SemanticException: Could not interpret path expression 'type_id'`